examples/demo-rollup/Cargo.toml

```diff
@@ -43,3 +43,5 @@ const-rollup-config = { path = "../const-rollup-config" }
 
 [dev-dependencies]
 tempfile = "3.5.0"
+blake3 = "1.3"
```

Comment:
nit: We've been using sha2 for testing in other places. Not a big deal at all, but it would be nice to stay consistent for now.